NEW-MEMBERS.md

```diff
@@ -0,0 +1,29 @@
+## New Members
+
+The purpose of this plan is to ensure that you become familiar with the team and
+know how you will contribute. The first step is to get yourself familiar with
+our mission at [Security TAG charter](governance/charter.md).
+
+New members are advised to:
+
+* Join the [CNCF Slack team](https://slack.cncf.io/), particularly
+  [#tag-security](https://cloud-native.slack.com/messages/CDJ7MLT8S) channel and
+  introduce yourself.
+* Initially go through the following documents in the repository:
+    * [README.md](README.md)
+    * [CODE-OF-CONDUCT.md][coc]
+    * [Use cases and personas][use-cases]
+* Regularly join one of the [Zoom meetings][meeting-times] at least for the first
+  couple of months to get yourself up to speed.
+* Here are multiple ways to get involved:
+    * Join the meeting as advised above and express your area of interests or if
+      you want to work on any specific issue.
+    * Express your thoughts or ask questions on an issue you find interesting.
+    * Choose an issue where [help is
+      needed](https://github.com/cncf/tag-security/labels/help%20wanted) and
```

Comment:
`/labels/help%20wanted` didn't work or this just was overlooked?

Reply:
I don't think that works because it's actually relative to something like `https://github.com/cncf/tag-security/blob/...` and I think it might be a bit fragile to use `../..`